custom-namespaces.yaml

```
@@ -162,6 +162,10 @@ contextual_services:
     - rburwei@mozilla.com
   spoke: looker-spoke-private
   views:
+    final_adM_forecasts:
+      type: ping_view
```

```suggestion
      type: table_view
```
Not sure what the structure of the spreadsheet is, but ping_views are usually used for ping tables and require a submission_date, client_id, and document_id column